### PR TITLE
Add close all logs option for verbose

### PR DESCRIPTION
**Pull request by kabeor**
### Which kind of PR do you create?

- [ ] This PR only contains minor fixes.
- [ ] This PR contains major feature update.
- [ ] This PR introduces a new function/api for Qiling Framework.

### Coding convention?

- [ ] The new code conforms to Qiling Framework naming convention.
- [ ] The imports are arranged properly.
- [ ] Essential comments are added.
- [ ] The reference of the new code is pointed out.

### Extra tests?

- [ ] No extra tests are needed for this PR.
- [ ] I have added enough tests for this PR.
- [ ] Tests will be added after some discussion and review.

### Changelog?

- [ ] This PR doesn't need to update Changelog.
- [ ] Changelog will be updated after some proper review.
- [ ] Changelog has been updated in my PR.

### Target branch?

- [ ] The target branch is dev branch.

### One last thing

- [ ] I have read the [contribution guide](https://docs.qiling.io/en/latest/contribution/)

-----

### CODE REVIEW

File: qiling/core.py [hunk 1]
```diff
@@ -547,6 +551,8 @@ def verbose(self):
     @verbose.setter
     def verbose(self, v):
         self._verbose = v
+        if self._verbose == -1:
+            self.log.disabled = True
```
Comment (wtdcode) [inline qiling/core.py:555]: `Logger.disabled` is undocumented attributed. Use `Logger.setLevel` instead. Besides, the if statement shall go to `ql_resolve_logger_level`.

File: qiling/const.py [hunk 2]
```diff
@@ -36,11 +36,12 @@ class QL_OS(IntEnum):
     BLOB = 210
 
 class QL_VERBOSE(IntEnum):
-    OFF = 0
-    DEFAULT = 1
-    DEBUG = 4
-    DISASM = 10
-    DUMP = 20
+    OFF = -1    # turn off all the output
+    MINIMUM = 0 # output only warnings
```
Comment (wtdcode) [inline qiling/const.py:40]: This is causing a break change. Keep the value of `OFF` and assign another value for `-1` instead.

**Comment by elicn** [comment]
I find `OFF` and `DISABLED` to be confusing.. and neither of them actually turns off logging altogether.
What about changing the existing `OFF` (not really off but show only warnings) to be `WARNING` and have `OFF` really disabling the logging (like `console=False` does)?

**Comment by kabeor** [comment]
like @wtdcode said “This is causing a break change.”

**Comment by elicn** [comment]
Can you elaborate what exactly gets broken, and because of what..?

**Comment by kabeor** [comment]
people's project which based on our code need change their verbose to adapt ours

**Comment by xwings** [comment]
Make sure you update the documents too

**Review by xwings — APPROVED** [review]

**Review by xwings — APPROVED** [review]

**Pull request merged**